File: project_euler/Problem025/Problem025.c
#include <stdio.h>
#include <math.h>

void Problem025(void) {
    int target = 1000;
    int result = (int) ceil(((double) target - 1 + log10(5) / 2) / log10((1 + sqrt(5)) / 2));
    printf("%d\n", result);
}

File: project_euler/Problem025/Problem025.js
function Problem025() {
    let target = 1000
    let result = Math.ceil((target - 1 + Math.log10(5) / 2) / Math.log10((1 + Math.sqrt(5)) / 2))
    console.log(result)
}

module.exports = {
    Problem025 : Problem025
};
